scalalib/src/mill/javalib/android/AndroidSdkModule.scala

```diff
@@ -66,6 +65,13 @@ trait AndroidSdkModule extends Module {
     PathRef(buildToolsPath().path / "aapt")
   }
 
+  /**
+   * Provides the path to AAPT2, used for resource handling and APK packaging.
```

Comment:
Is there any `aapt2` documentation we can link to here?

Reply:
Yes i will add

Reply:
?